worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -331,18 +330,13 @@ private boolean isOptOutPatient(String patientId) {
                 .anyMatch(optOut -> optOut.getEffectiveDate().isBefore(tomorrow));
     }
 
-    private int processHandles(List<Future<Integer>> futureHandles, ProgressTracker progressTracker) {
-        int errorCount = 0;
-
+    private void processHandles(List<Future<Void>> futureHandles, ProgressTracker progressTracker) {
         var iterator = futureHandles.iterator();
         while (iterator.hasNext()) {
             var future = iterator.next();
             if (future.isDone()) {
                 try {
-                    var responseCount = future.get();
-                    if (responseCount > 0) {
-                        errorCount += responseCount;
-                    }
+                    future.get();
```

Comment:
Curious why this approach was chosen of periodically sending a list to the processHandles method vs. using the API for all futures to complete like https://stackoverflow.com/a/51006865/12265284

Reply:
Well, in that case, all the calls are made in a burst mode at the outset and it just needs to await for all of them to return and join. Since our batch could run for a very long time, we do some work periodically (eg track process etc).

Reply:
Ahh ok, was trying to think of ways to reduce complexity by possibly removing the 2nd call to processHandles